packages/react-composites/src/composites/CallComposite/styles/Buttons.styles.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { concatStyleSets, IButtonStyles, IContextualMenuItemStyles, IStyle } from '@fluentui/react';
+import { DevicesButtonStyles, ParticipantsButtonStyles } from '@internal/react-components';
+
+const MINIMUM_TOUCH_TARGET_HEIGHT_REM = 3;
+
+/** @private */
+export const controlBarContainerStyles: IStyle = {
+  paddingTop: '0.25rem',
+  paddingBottom: '0.25rem',
+
+  // @TODO: this should be exposed through a custom CallComposite Theme API that extends the fluent theme with semantic values
+  boxShadow: `
+    0px 6.400000095367432px 14.399999618530273px 0px #00000021;
+    0px 1.2000000476837158px 3.5999999046325684px 0px #0000001A;
+  `
+};
+
+/**
+ * @private
+ */
+export const groupCallLeaveButtonStyle = {
+  root: {
+    border: '0.125rem',
+    borderRadius: 2,
```

Comment:
Let's not change it in this PR. But maybe we can have a follow up cleanup PR for converting hardocded border radius and box shadows to use `theme.effects.roundedCorner` and `theme.effects.elevation`